src/pages-and-resources/discussions/app-config-form/apps/lti/LtiConfigForm.jsx

```diff
@@ -1,23 +1,26 @@
 import React, { useEffect } from 'react';
 import PropTypes from 'prop-types';
 
-import { injectIntl, intlShape } from '@edx/frontend-platform/i18n';
-import {
-  Card, Form,
-} from '@edx/paragon';
+import { ensureConfig, getConfig } from '@edx/frontend-platform';
+import { getAuthenticatedUser } from '@edx/frontend-platform/auth';
+import { FormattedMessage, injectIntl, intlShape } from '@edx/frontend-platform/i18n';
+import { Card, Form, MailtoLink } from '@edx/paragon';
 
 import { useFormik } from 'formik';
-import * as Yup from 'yup';
 import { useDispatch } from 'react-redux';
+import * as Yup from 'yup';
 
-import {
-  updateValidationStatus,
-} from '../../../data/slice';
+import { updateValidationStatus } from '../../../data/slice';
 import AppExternalLinks from '../shared/AppExternalLinks';
 import messages from './messages';
 
+ensureConfig([
+  'SITE_NAME',
+  'SUPPORT_EMAIL',
+], 'LTI Config Form');
+
 function LtiConfigForm({
-  appConfig, app, onSubmit, intl, formRef, title,
+  appConfig, app, onSubmit, intl, formRef, providerName,
```

Comment:
`title` is a bit ambiguous here so changed it to what it really contains, the provider name.